content/03.methods.md

```diff
@@ -13,49 +13,52 @@ Preprints on bioRxiv are versioned, and in our snapshot, 26,905 out of 98,023 co
 When preprints had multiple versions, we used the latest one unless otherwise noted.
 Authors submitting preprints to bioRxiv can select one of twenty-nine different categories and tag the type of article: a new result, confirmatory finding, or contradictory finding.
 A few preprints in this snapshot were later withdrawn from bioRxiv; when withdrawn, their content is replaced with the reason for withdrawal.
-As there were very few withdrawn preprints, we did not treat these as a special case.
+We encountered a total of 72 withdrawn preprints within our snapshot.
```

Comment:
I'm guessing this changes all the results figures extremely subtly - did you update all of them for this change?